$NetBSD: patch-Makefile,v 1.1 2019/03/24 20:50:10 thorpej Exp $

u-boot: Fix name of python2 interpreter when built in pkgsrc environment

--- Makefile.orig	2019-03-24 19:18:59.903169960 +0000
+++ Makefile
@@ -359,7 +359,7 @@ YACC		= bison
 AWK		= awk
 PERL		= perl
 PYTHON		?= python
-PYTHON2		= python2
+PYTHON2		= python2.7
 PYTHON3		= python3
 DTC		?= $(objtree)/scripts/dtc/dtc
 CHECK		= sparse
